config/rbac/role_binding.yaml

```diff
@@ -1,4 +1,17 @@
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: manager-clusterrolebinding
```

Comment:
Might want to rename this to something more descriptive.

Reply:
Yep, see my other comment.